tests/system/conftest.py

```diff
@@ -128,7 +128,10 @@ def resourcemanager_client(
 
 @pytest.fixture(scope="session")
 def session() -> bigframes.Session:
-    return bigframes.Session()
+    context = bigframes.BigQueryOptions(
+        location="US",
```

Comment:
This should already be the default https://github.com/googleapis/python-bigquery-dataframes/blob/c67e501a4958ac097216cc1c0a9d5c1530c87ae5/bigframes/session/__init__.py#L182

Reply:
Note that the line below that raises a warning. So this avoids that noise at least, even if it doesn't fix the issue.